nltools/data/brain_data.py

```diff
@@ -350,15 +362,31 @@ def to_nifti(self):
 
         return self.nifti_masker.inverse_transform(self.data)
 
-    def write(self, file_name=None):
-        """ Write out Brain_Data object to Nifti File.
+    def write(self, file_name=None, compression='blosc'):
```

Comment:
I know this is WIP, but compression should refer to either nifti or hdf5. Probably don't want the default to be blosc.  An alternative approach would be to make compression a kwarg argument and if compression isn't in the kwarg list to default it to 'blosc'.  As long as this is in the docstrings it will be less likely to lead to confusion for writing out nifti images.

Reply:
Went with the second proposal here and just made a kwargs which deepdish will look for or otherwise default to 'blosc'